test/test.el

```diff
@@ -533,6 +533,32 @@ buffer and do not save the results."
            :groups '((:pred (lambda (item)
                               (s-contains? "moon" item)))))))
 
+(ert-deftest org-super-agenda--test-:property-list-val ()
+  ;; DONE: Works.
+  (should (org-super-agenda--test-run
+           :groups '((:property ("Effort" "5" ))))))
+
+(ert-deftest org-super-agenda--test-:property-list-fun ()
+  ;; DONE: Works.
+  (should (org-super-agenda--test-run
+           :groups '((:property ("Effort" (lambda (v) (string= v "5"))))))))
+
+(ert-deftest org-super-agenda--test-:property-str ()
+  ;; DONE: Works.
+  (should (org-super-agenda--test-run
+           :groups '((:property "Effort")))))
+
+(ert-deftest org-super-agenda--test-:property-list ()
+  ;; DONE: Works.
+  (should (org-super-agenda--test-run
+           :groups '((:property ("Effort"))))))
+
+(ert-deftest org-super-agenda--test-:property-err-check ()
+  ;; DONE: Works.
+  (let ((debug-on-error nil))
+    (should-error (org-super-agenda--test-run
+                   :groups '((:property ("Effort" 'sym-not-allowed-here)))))))
```

Comment:
Instead of testing a non-existent function, let's test with one that does exist, like `stringp`.  Note that this list is already quoted, so the symbol shouldn't be quoted again.

If it seems important enough, we can allow for `should-error` forms and test a non-function symbol later.

Reply:
I just removed it for now - it was not so important. I think I just wanted to write a test to see if the error was thrown correctly.